modules/Blockchain/Ethereum/desktop.ini

```diff
@@ -1,6 +0,0 @@
-[.ShellClassInfo]
-IconResource=C:\Users\Uros\Pictures\Icons\PRY set\Pry Black\Industrial Black.ico,0
```

Comment:
Is it safe to remove this file?

Reply:
I think so, it is dummy .ini file from Uki's PC